kafka-cron/utils/assert_command.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/berkeli/kafka-cron/types"
+)
+
+func AssertCommandsEqual(t *testing.T, want, got types.Command) error {
```

Comment:
"Assert" as a prefix to a function name strongly suggests that if things aren't equal, the test will be failed via a call to `t.Fatal` or `t.Error` - I'd suggest either making that be the case, or renaming the function.